cluster/terraform_kubernetes/ingress_controller.tf

```diff
@@ -212,6 +212,223 @@ resource "helm_release" "ingress-nginx" {
   }
 }
 
+resource "helm_release" "ingress-nginx-clone" {
```

Comment:
It's difficult to maintain if we must repeat all config items. Could we use the technique we had before? https://github.com/DFE-Digital/teacher-services-cloud/pull/278/files#diff-bde884c18cdde6ad4a4bc94bc5a823630c3349c19c34b4579f8feb8802dd6aaaL171
We deleted it as it was not tested for a long time